### PR TITLE
feat: update staking contract addresses

### PR DESCRIPTION
**Pull request by Tanya-atatakai**
## Proposed changes

Update meme staking contract and its activity checker addresses

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: frontend/config/activityCheckers.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const OPTIMISM_ACTIVITY_CHECKERS: ActivityCheckers = {};
 
 export const BASE_ACTIVITY_CHECKERS: ActivityCheckers = {
   [ActivityCheckerType.MemeActivityChecker]: new MulticallContract(
-    '0xAe2f766506F6BDF740Cc348a90139EF317Fa7Faf',
+    '0x026AB1c5ea14E61f67d245685D9561c0c2Cb39Ba',
```
Comment (Tanya-atatakai) [inline frontend/config/activityCheckers.ts:36]: https://basescan.org/address/0xc653622FD75026a020995a1d8c8651316cBBc4dA#readProxyContract#F2

File: frontend/config/stakingPrograms/base.ts [hunk 2]
```diff
@@ -12,12 +12,12 @@ import { StakingProgramMap } from '.';
 
 export const BASE_STAKING_PROGRAMS_CONTRACT_ADDRESSES: Record<string, Address> =
   {
-    [StakingProgramId.MemeBaseAlpha]:
-      '0x06702a05312091013fdb50c8b60b98ca30762931',
+    [StakingProgramId.MemeBaseAlpha2]:
+      '0xc653622FD75026a020995a1d8c8651316cBBc4dA',
```
Comment (Tanya-atatakai) [inline frontend/config/stakingPrograms/base.ts:16]: https://basescan.org/address/0xc653622FD75026a020995a1d8c8651316cBBc4dA

File: frontend/config/stakingPrograms/base.ts [hunk 3]
```diff
@@ -12,12 +12,12 @@ import { StakingProgramMap } from '.';
 
 export const BASE_STAKING_PROGRAMS_CONTRACT_ADDRESSES: Record<string, Address> =
   {
-    [StakingProgramId.MemeBaseAlpha]:
-      '0x06702a05312091013fdb50c8b60b98ca30762931',
+    [StakingProgramId.MemeBaseAlpha2]:
```
Comment (Tanya-atatakai) [inline frontend/config/stakingPrograms/base.ts:15]: the name is from here https://gateway.autonolas.tech/ipfs/f01701220a4281ec71a254c8dd08d624ff09609dc3235a0e63b05c1e410eb7e1f3d07e57d

File: operate/ledger/profiles.py [hunk 4]
```diff
@@ -91,7 +91,7 @@
         "optimus_alpha": "0x88996bbdE7f982D93214881756840cE2c77C4992",
     },
     Chain.ETHEREUM: {},
-    Chain.BASE: {"meme_base_alpha": "0x06702A05312091013FdB50C8b60B98ca30762931"},
+    Chain.BASE: {"meme_base_alpha_2": "0xc653622FD75026a020995a1d8c8651316cBBc4dA"},
```
Comment (Tanya-atatakai) [inline operate/ledger/profiles.py:94]: @jmoreira-valory not sure if the name is shared between FE and BE only or it should be updated in the service/agent/something else as well?

Comment (truemiller) [reply operate/ledger/profiles.py:94]: @Tanya-atatakai it is shared between both, https://github.com/valory-xyz/olas-operate-app/blob/87eea7dfccedcf0c1e01ab3d83c2621e611056c7/frontend/service/Services.ts#L78

**Review by truemiller — APPROVED** [review]

File: operate/ledger/profiles.py [hunk 5]
```diff
@@ -91,7 +91,7 @@
...
     Chain.CELO: {
         "meme_celo_alpha": "0xf39cd0eE4C502Df7D26F28cFAdd579724A3CFCe8",
```
Comment (kupermind) [inline operate/ledger/profiles.py:96]: 0x95D12D193d466237Bc1E92a1a7756e4264f574AB

Comment (kupermind) [reply operate/ledger/profiles.py:96]: activity checker:
https://celoscan.io/address/0x95D12D193d466237Bc1E92a1a7756e4264f574AB#readProxyContract#F2

**Review by kupermind — APPROVED** [review]

File: operate/ledger/profiles.py (hunk 4 above)
Comment (jmoreira-valory) [reply operate/ledger/profiles.py:94]: @Tanya-atatakai The staking contract ID (e.g., meme_base_alpha_2) is passed to the backend only in the service template when creating/updating a service. Based on that value, the backend will fill a number of env variables ("COMPUTED") that deppend on the particular staking contract.

Please see comment above from @truemiller regarding the frontend.

**Pull request merged**